### ISSUE TITLE
Confuse about the dataset split

### ISSUE THREAD

**Issue by zyzisastudyreallyhardguy** [amazon-research/gnn-tail-generalization#6]
Dear authors,

Could you please provide the detailed training/val/test split you adopt in your paper? 
For example, for Cora, in your code, it seems you use the first 600 nodes as the training set while using the same validation and test set of the public split. But this may conflict with the isolation split you adopt (10% of lowest degree nodes) as there may be overlapped nodes between the isolation nodes and the training set.

Thus, how do you conduct the evaluation on isolation split? Did you ignore all the isolation nodes in the training set during the evaluation?

Looking forward to your reply! Many Thanks！
